Documentation/gettingstarted/k8s-install-kubespray.rst

```diff
@@ -141,15 +141,24 @@ Installing Kubernetes cluster with Cilium as CNI
 
 Kubespray uses Ansible as its substrate for provisioning and orchestration. Once the infrastructure is created, you can run the Ansible playbook to install Kubernetes and all the required dependencies. Execute the below command in the kubespray clone repo, providing the correct path of the AWS EC2 ssh private key in ``ansible_ssh_private_key_file=<path to EC2 SSH private key file>``
 
-We recommend using the `latest released Cilium version`_ by editing ``roles/download/defaults/main.yml``. Open the file, search for ``cilium_version``, and replace the version with the latest released. As an example, the updated version entry will look like: ``cilium_version: "v1.2.0"``.
-
+We recommend using the `latest released Cilium version`_ by passing the variable when running the ansible-playbook command.
```

Comment:
Spell checking complains on `ansible` on this line. We have `Ansible` already in the list of acceptable words, can you please turn it to lowercase so it's not dependent on case? Just this:

```
$ sed -i 's/Ansible/ansible/' Documentation/spelling_wordlist.txt
```

Reply:
I don't know how I missed that :) Added backticks to "ansible-playbook"

Also changed `Ansible` into `ansible` and placed it properly.

Reply:
GitHub workflows need a manual approval to run on PRs from first-time contributors, so you didn't get the results from spell checking (or from any other test) on creating the PR.

Thanks for the update!